packages/blade/src/components/Form/Selector/SelectorInput.web.tsx

```diff
@@ -28,8 +29,13 @@ const getHoverStyles = ({
   const borderColor = hoverTokens.default.border?.[checked];
 
   return {
-    borderColor: borderColor ? getIn(theme, borderColor) : undefined,
-    backgroundColor: getIn(theme, backgroundColor),
+    borderColor: borderColor
+      ? borderColor === 'transparent'
+        ? 'transparent'
+        : getIn(theme, borderColor)
+      : undefined,
+    backgroundColor:
```

Comment:
We can remove this code and replace this with just `getIn(theme, borderColor)` because in our theme we now have a token called `transparent` which can be accessed via `colors.transparent`

```
    borderColor: getIn(theme, borderColor!),
    backgroundColor: getIn(theme, backgroundColor),
```

And in `packages/blade/src/components/Form/Selector/types.ts`

```diff
- type ColorTokens = `colors.${DotNotationToken<Theme['colors']>}` | 'transparent';
+ type ColorTokens = `colors.${DotNotationToken<Theme['colors']>}`;

```

Reply:
Nice! This makes things easy!